ctapipe/reco/reconstructor.py

```diff
@@ -66,6 +90,34 @@ def __call__(self, event: ArrayEventContainer):
             reconstructed stereo geometry and telescope-wise impact position.
         """
 
+    @classmethod
+    def read(cls, path, **kwargs):
```

Comment:
add a docstring here to explain what this does aand what is expected from `path`.   It is  not obvious that path should be an instance of this class pickled with joblib for example.

Reply:
done